src/components/FindWallet/MobileFiltersMenu.tsx

```diff
@@ -47,7 +43,6 @@ export const MobileFiltersMenu = ({
   onClose,
 }: MobileFiltersMenuProps) => {
   const { t } = useTranslation("page-wallets-find-wallet")
-  const { filteredWallets } = useWalletTable({ filters, t, walletData })
```

Comment:
Moved this hook to the parent (page component) to share the results and minimize the calls we do for it since its doing the main filtering.